### ISSUE TITLE
Point the slicebox:latest Docker image to version 1.4.2

### ISSUE THREAD

**Issue by ghisvail** [slicebox/slicebox#273]
The current `latest` image is still pointing to version `1.4.1`. I believe it should be pointing to version `1.4.2` now, right?

**Comment by ghisvail** [comment]
Also, since you guys are using `alpine` only (via the openjdk image I guess), is it really necessary to add it as a suffix to the name of the Docker image? Since no alternatives are provided (no `slicebox/slicebox:latest-ubuntu`) it seems a bit pointless.

Same comment regarding the `latest` Docker image. Since version `1.4` is the only version maintained, and version `2.0` remains in the making, I don't see the point of explicitly pulling `slicebox/slicebox:1.4-alpine-latest`.

In a nutshell, could we just have `slicebox/slicebox:latest` as the Docker image pointing to the latest supported version?

**Comment by kobmic** [comment]
Hi @ghisvail
I agree, so I just created the tag "slicebox/slicebox:latest" on dockerhub (pointing to same image as 1.4.2-alpine). I still want to keep naming "alpine" "ubuntu" etc. even if we don't have ubuntu yet. But we might consider to change/add base images when going to jre 9 or 10 as these are not available on alpine at the moment. But I don't think we need the labels "1.4.-linux-flavour-latest". 
@KarlSjostrand: if you agree, could you remove tag "1.4-alpine-latest", you're the owner of the dockerhub repo, I think I don't have the right to remove.

**Comment by ghisvail** [comment]
> I just created the tag "slicebox/slicebox:latest" on dockerhub

Awesome, thanks!
